cfme/containers/container.py

```diff
@@ -8,7 +8,7 @@
                                       ContainerObjectDetailsBaseView,
                                       GetRandomInstancesMixin, Labelable, LoggingableView)
 from cfme.exceptions import ItemNotFound
-from cfme.common import Taggable, TagPageView
+from cfme.common import CustomButtonEventsMixin, Taggable, TagPageView
```

Comment:
Just curious, can you explain why this Mixin is necessary? So that each of the entities have access to `get_button_events`?

Reply:
@john-dupuy yup, Custom button events are common for overall cfme. To avoid repetition of `navigation`, `views` and `methods` for each entities related to custom button.